### PR TITLE
Refactor code

### PR DESCRIPTION
**Pull request by darlaevans2000**
# Description
Refactored code to make it look clean and removed instructions that were added by instructors

## Type of change

- [X ] Refactor (non-breaking change which fixes wet code)

# How Has This Been Tested?
open index.html

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -7,29 +7,26 @@ var form = document.querySelector(".form-view");
...
-  // new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
+   new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
 ];
 var currentCover = {};
 
-
-// Add your event listeners here 👇
+//event listeners
 window.addEventListener("load", randomBook);
 randomButton.addEventListener("click", randomBook);
```
Comment (Kristiannmiller) [inline src/main.js:31]: The functionality works great here, but I think you could name the randomBook function and formVisibility function a little more clearly. Naming functions after the action they perform works best for coder empathy. For example: createRandomCover or displayRandomCover would help give a little more insight into what the function is doing.

File: src/main.js [hunk 2]
```diff
@@ -7,29 +7,26 @@ var form = document.querySelector(".form-view");
...
 var savedCovers = [
-  // new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
+   new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
 ];
 var currentCover = {};
 
-
-// Add your event listeners here 👇
+//event listeners
 window.addEventListener("load", randomBook);
```
Comment (Kristiannmiller) [inline src/main.js:30]: This works! Another way you could do this is

window.onload = randomBook();

Same deal, just thought you might like to see an alternative way as well.

File: src/main.js [hunk 3]
```diff
@@ -74,7 +66,7 @@ function formVisibility() {
   savedViewSection.classList.add("hidden");
 };
 
-function savedPosterGridView() {
+function savedCoverGridView() {
```
Comment (Kristiannmiller) [inline src/main.js:69]: There seems to be a lot of adding and remove the "hidden" attribute from classes to show or hide pages. It works great the way you have it, but this repetition makes me wonder if there's a good opportunity to refactor a few functions into one collecting changePageView function using a parameter to establish which page to display.

File: src/main.js [hunk 4]
```diff
@@ -105,22 +97,22 @@ function homeVisibility() {
...
+  covers.push(coverField.value);
+  titles.push(titleField.value);
+  descriptors.push(desc1Field.value, desc2Field.value);
+  coverImg.setAttribute("src", coverField.value);
+  title.innerText = titleField.value;
+  descriptor1.innerText = desc1Field.value;
+  descriptor2.innerText = desc2Field.value;
+};
+
 function createNewBook(event) {
```
Comment (Kristiannmiller) [inline src/main.js:110]: Try to stick with consistent language when naming functions as well. I've seen both "cover" and "book" used to describe the same thing. Might get a bit confusing

File: src/main.js [hunk 5]
```diff
@@ -105,22 +97,22 @@ function homeVisibility() {
...
-    descriptors.push(desc1Value, desc2Value);
-    currentCover = new Cover(coverValue, titleValue, desc1Value, desc2Value);
-    coverImg.setAttribute("src", coverValue);
-    title.innerText = titleValue;
-    descriptor1.innerText = desc1Value;
-    descriptor2.innerText = desc2Value;
+    pushAndSetFieldValues();
+    currentCover = new Cover(coverField.value, titleField.value, desc1Field.value, desc2Field.value);
   } else {
     alert("Please fill out the full form");
```
Comment (Kristiannmiller) [inline src/main.js:117]: Nice error handling!

**Review by Kristiannmiller — APPROVED** [review]
This looks really good, and functionality is all there! I would go through and work on some function names, and maybe try to refactor page viewing into one function. After that just make sure everything is understood and can be explained by each of you. 

Bonus - your readme is amazing!!! Never underestimate the power of an awesome Readme! 

Good work! It looks great!

File: src/main.js (hunk 1 above)
Comment (hoomberto) [reply src/main.js:31]: Great points on incorporating coder empathy! We'll definitely have to make some cleaner names for these

File: src/main.js [hunk 6]
```diff
@@ -105,22 +97,22 @@ function homeVisibility() {
...
+
 function createNewBook(event) {
   event.preventDefault();
-  var coverValue = coverField.value;
-  var titleValue = titleField.value;
-  var desc1Value = desc1Field.value;
-  var desc2Value = desc2Field.value;
-  if (coverValue && titleValue && desc1Value && desc2Value) {
+  if (coverField.value && titleField.value && desc1Field.value && desc2Field.value) {
     homeVisibility();
```
Comment (hoomberto) [inline src/main.js:113]: In the next refactoring, let's rename anything with visibility to something more specific, i.e. show, or hide in the name.

**Review by hoomberto — APPROVED** [review]

**Pull request merged**